### PR TITLE
Rebuild specs for Pods-resources.sh change

### PR DESCRIPTION
**Pull request by postmechanical**
Required for https://github.com/CocoaPods/CocoaPods/pull/6618

### CODE REVIEW

**Comment by postmechanical** [comment]
@dnkoutso 👀 🙏

File: init_single_platform/after/SampleApp.xcodeproj [hunk 1]
```diff
@@ -44,15 +44,15 @@ Targets:
           GCC_PRECOMPILE_PREFIX_HEADER: 'YES'
           GCC_PREFIX_HEADER: SampleApp/SampleApp-Prefix.pch
           INFOPLIST_FILE: SampleApp/SampleApp-Info.plist
-          PRODUCT_NAME: $(TARGET_NAME)
+          PRODUCT_NAME: "$(TARGET_NAME)"
```
Comment (dnkoutso) [inline init_single_platform/after/SampleApp.xcodeproj]: did those change as part of your change? Sometimes I've had this happen to me, my advise is to checkout all *.xcodeproj changs and just update the `.sh` files

Comment (postmechanical) [inline init_single_platform/after/SampleApp.xcodeproj]: @dnkoutso Yes, those changed as a result of running `bundle exec rake spec:rebuild_integration_fixtures`

Comment (segiddins) [inline init_single_platform/after/SampleApp.xcodeproj]: it's ruby / psych version dependent 😭

Comment (dnkoutso) [inline init_single_platform/after/SampleApp.xcodeproj]: @postmechanical just revert the `*.xcodeproj` changes we can merge the remaining changes.

Comment (postmechanical) [inline init_single_platform/after/SampleApp.xcodeproj]: What about the quote only changes to the lock files?

Comment (dnkoutso) [inline init_single_platform/after/SampleApp.xcodeproj]: revert them. I am fairly certain you only need `.sh` file changes based of your shell script change PR.

**Comment by postmechanical** [comment]
@dnkoutso I reverted all the quote only changes.

**Comment by dnkoutso** [comment]
@postmechanical awesome! Merging.

**Comment by dnkoutso** [comment]
@postmechanical now rebase your branch and it should be green.

**Pull request merged**